build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.1.3"
+        classpath 'com.android.tools.build:gradle:4.1.2'
         classpath 'com.google.gms:google-services:4.3.5'
         classpath 'com.google.dagger:hilt-android-gradle-plugin:2.33-beta'
 
```

Comment:
Should this be in the PR ? Especially since it's downgrading the version....

Reply:
I might have made a mistake here you are right